tests/test_pdf.py

```diff
@@ -5,6 +5,26 @@
 import json
 
 
+def test_hello_world_ctor():
```

Comment:
I think I'm tired, but what is `ctor`?

Reply:
constructor

Reply:
Thanks. :+1: Much more obvious now after sleep, my bad.